src/state/store/initialState.js

```diff
@@ -1,9 +1,11 @@
 const initialState = {
   error: false,
   articles: [],
+  backyardArticles: [],
   article: {},
   subscriber: false,
   open: false,
+  backyardArticle: [],
```

Comment:
Shouldn't a single backyard article be an object like article?

Reply:
yes, its fixed now!